core/src/main/scala/org/broadinstitute/dsde/rawls/entities/base/EntityProvider.scala

```diff
@@ -34,9 +34,13 @@ trait EntityProvider {
 
   // ----- implementation methods follow:
 
-  def batchUpdateEntities(entityUpdates: Seq[EntityUpdateDefinition], parentContext: RawlsRequestContext): Future[Traversable[Entity]]
+  def batchUpdateEntities(entityUpdates: Seq[EntityUpdateDefinition],
+                          parentContext: RawlsRequestContext
+  ): Future[Traversable[Entity]]
```

Comment:
not sure why this PR has so much reformatting; this happened after rebasing #3249 into here.